### DISCUSSION TITLE
How do I set ASS Video Resolution to default Zero?

### DISCUSSION THREAD

**Discussion by dmaccountsetup** [SubtitleEdit/subtitleedit#6767] [Q&A] [answered]
When I am working with ASS subs, it automatically adjust the size of the font according to the resolution of the video...for example, if my video is 1080p, it adjusted to size 75 (instead of original size 20). The font size may not be a big issue for most players but I use PLEX player on Amazon 4K Firestick (streaming media from home network server) and it blows up to the actual size 75 taken up 50% of my TV screen. If I change the res to ZeroxZero under properties, the size won't adjusted back to original size. How to I set the resolution to 0x0 for ASS so that it won't mess up the size of the original fonts?
Thank you!

**Comment by niksedk** [comment, marked as answer]
The default video resolution for ASSA files is 384x288 - that's why font size 20 is not super tiny when not having `PlayResX` and `PlayResY` in the `[Script Info]` header (as the font size is up-scaled to match the actual video resolution).

It makes no sense to have `PlayResX` and `PlayResY` as zero - it's better to remove them or use the default resolution.
Something in your setup does not handle ASSA files correctly, so perhaps it's better to use SubRip .srt files.

In the file `Settings.xml` you can probably change these to false, to avoid SE adding `PlayResX` and `PlayResY` automatically:
````xml
    <AssaResolutionAutoNew>True</AssaResolutionAutoNew>
    <AssaResolutionPromptChange>True</AssaResolutionPromptChange>

````

**Reply by dmaccountsetup** [reply]
Thank you for the answer. The auto re-sizing won't be a problem anymore because I just found out that PLEX has fixed the ASS sub display issue I've been having. So, I am all set. Also, Setting the `AssaResolutionAutoNew`, `AssaResolutionPromptChange` values to False in `Setting.xml` did fix the problem.
For the purpose of our discussion earlier, I know Zero resolution doesn't sound logical but it is the easiest workaround which actually works (when entered Zero, it adjusted to default resolution like you mentioned). I could use other simpler formats like .srt but .srt wouldn't be able to handle something like in the screenshot (multiple characters,fonts, styles, positions, etc). Also, when it automatically resizing the font, it only does it to size but bordering and shading remain the same. For someone who like thick borders, it's a pain to go back and fix manually all borders and shadings. 
![vlcsnap-2023-03-22-10h42m02s295](https://user-images.githubusercontent.com/36719074/226994105-27db1020-ec9a-4055-996a-aa6e0dfe65e7.png)
